lib/src/bit_pattern.dart

```diff
@@ -62,7 +62,7 @@ abstract class BitPatternBuilder {
   ///
   /// print(pattern.match(0xD /* 0b1101 */)); // [1]
   /// ```
-  BitPattern<List<int>> build([String name]);
+  BitPattern<List<int>?> build([String? name]);
```

Comment:
Thanks. I think we don't want a `BitPattern` with `null`. I've made some changes.